app/controllers/links_controller.rb

```diff
@@ -9,6 +9,7 @@ def new
   def index
     @links = Link.all
     @presenter = Links::IndexPresenter.new
+    @user_links = current_user.links.order(created_at: :desc) if logged_in?
```

Comment:
Keep `@user_links` in `index` action since presenter has no access of `current_user`.